cmd/create/cluster/cmd.go

```diff
@@ -1759,14 +1758,14 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	fips := args.fips || fedramp.Enabled()
-	if interactive.Enabled() && fips && !fedramp.Enabled() {
```

Comment:
@vkareh what does this statement do? If we're removing the `fips` check, does that mean we _always_ enable FIPS? It should still be a use option, with the default to be False. It's _entirely_ possible and likely I'm just missing some other context ¯\_(ツ)_/¯

Reply:
This extra `fips` check was put in place so that the interactive "Do you want FIPS?" question showed up _only_ if the user manually specified the `--fips` flag. I removed it so that now any time the user is in interactive mode, the question will show up.

It doesn't change the actual value of the flag, just whether the interactive prompt shows.

Reply:
Oh, perfect! That was the context I was missing, thanks!